builder/proxmox/config.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 
 	Memory         int          `mapstructure:"memory"`
 	Cores          int          `mapstructure:"cores"`
+  CPUType        string       `mapstructure:"cpu_type"`
```

Comment:
Woops; I failed fixing this 🙂 
```suggestion
	CPUType        string       `mapstructure:"cpu_type"`
```

Reply:
Weird; I was allowed to fix the conflict; but not to commit this.